auto_rx/autorx/static/js/scan_chart.js

```diff
@@ -16,6 +16,9 @@ function setup_scan_chart(){
 	        ['x_spectra',autorx_config.min_freq, autorx_config.max_freq],
 	        ['Spectra',0,0]
 	    ],
+		colors: {
+			Spectra: "#2f77b4"
+		},
```

Comment:
Removing the legend had the side effect of assigning colours to the datasets randomly, so I've explicitly defined them here.